tools/js/wasm-themis/keygen.js

```diff
@@ -1,7 +1,7 @@
-const fs = require('fs')
-const themis = require('wasm-themis')
+var fs = require('fs')
```

Comment:
why changed const/let -> var?

Reply:
To give it more “ES5-ish” feel in `*.js` files, while `*.mjs` look more like ES6. Though again, it does not matter that much for Node (and these tools are intended to be run with Node).